src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -130,6 +130,14 @@
     <value>Invalid file name: It must contain only {0}.</value>
     <comment>{0} - The regular expression that the file name must match</comment>
   </data>
+  <data name="APT0004" xml:space="preserve">
+    <value>Invalid directory name. Directory names must not match the pattern '{0}'.</value>
+    <comment>{0} - The regular expression that the directory name must *not* match.</comment>
+  </data>
+  <data name="APT0005" xml:space="preserve">
+    <value>Invalid file name. File names must not match the pattern '{0}'.</value>
```

Comment:
Should we mention `AndroidResouce` here? as its specific that that ItemGroup cannot contain those items.

Reply:
This ignores the files silently now, so we don't need a new error code.